### ISSUE TITLE
blacklisted namespace causing reload

### ISSUE THREAD

**Issue by EFX-PXT1** [haproxytech/kubernetes-ingress#292]
running 1.5.3 with a pair of ingresses 'h' and 'p' deployed into same namespace.
each has ingressClasss defined (so 'h' or 'p').
'h' is set to blacklist an application namespace in which an ingress / service /  pod is defined for 'p'.
now if I repeatedly add/remove the application ingress in the blacklisted applcation namespace for 'p',
I was expecting no change on 'h'.
what I observe is that 'h' is impacted.
 
```
2021/03/17 18:59:15 WARNING events.go:206 Ingress 'delta-echo-ingress' not registered with controller, cannot delete !
2021/03/17 18:59:15 WARNING events.go:206 Ingress 'delta-echo-ingress' not registered with controller, cannot delete !
2021/03/17 18:59:21 TRACE   controller.go:174 HAProxy config sync started
...
2021/03/17 18:59:21 INFO    controller.go:306 HAProxy reloaded
2021/03/17 18:59:21 TRACE   controller.go:310 HAProxy config sync ended
[WARNING] 075/185921 (38) : Reexecuting Master process
```

am I mis-understanding the purpose of the whitelist/blacklist or tripping over something else ?
the aspiration is to have separate ingresses in a multi-tenant cluster which do not interfere.

**Comment by Mo3m3n** [comment]
Hi 
This is was probably fixed here https://github.com/haproxytech/kubernetes-ingress/commit/01fb294929b6e896893e9571164b9b9fd9495ad7 which is now available in `1.5.4`
I was able to reproduce your issue before the fix and after the patch it seems to be fixed.

**Comment by EFX-PXT1** [comment]
Many thanks @Mo3m3n - 1.5.4 does indeed resolve my issue. Many Thanks.
